### ISSUE TITLE
No commits for path: ., skipping

### ISSUE THREAD

**Issue by sashafklein** [google-github-actions/release-please-action#534]
### TL;DR

I setup release-please using v3 and Github Actions, and the action appears to run properly, but nothing happens. There are definitely correctly-formatted commits in my main app between the most recent and the specified bootstrap sha.

### Expected behavior

I was expected a release PR to be generated.

### Observed behavior

Nothing appears to happen after a "successful" GH Actions run (see below for output).

### Action YAML

```yaml
# https://github.com/marketplace/actions/release-please-action
on:
  push:
    branches:
      - release-please-test
name: "Orchestrate version releases"
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          release-type: node
          package-name: my-package-name
```

### Log output

```txt
Run google-github-actions/release-please-action@v3
✔ Looking for latest release on branch: master with prefix: my-package-name
⚠ Could not find releases.
✔ Building releases
✔ Building strategies by path
✔ Looking for latest release on branch: master with prefix: my-package-name
⚠ Could not find releases.
✔ Building pull requests
✔ Building strategies by path
✔ Collecting release commit SHAs
⚠ Could not find releases.
⚠ Expected 1 releases, only found 0
⚠ Missing 1 paths: .
✔ Collecting commits since all latest releases
⚠ Release SHA 39d4899dbdf12078c1d844c590f7e318f1a9b6cc did not have an associated pull request
✔ Splitting 476 commits by path
⚠ No latest release pull request found.
✔ Building candidate release pull request for path: .
✔ Considering: 0 commits
✔ No commits for path: ., skipping
```

### Additional information

My current setup is a simple. Node app I did one manual release while debugging this stuff. I then tried deleting it to fix this, but that didn't help, and the PR is, of course, still tagged with the release #.

I've been testing this by progressively changing the non-master branch that release-please should run on, to get it figured out before merging into master. So the `release-please-test` branch specified above was just one test. (`cltest` etc below are others.)

My `release-please-config.json`:

```json
{
  "bootstrap-sha": "39d4899dbdf12078c1d844c590f7e318f1a9b6cc",
  "packages": {
    ".": {
      "release-type": "node"
    }
  }
}
```

My `.release-please-manifest.json`:

```json
{
  ".": "1.0.22"
}
```

The last commits until the specified sha:

```
d39c897 - (HEAD -> commit-lint, origin/commit-lint) fix: trying wo release (21 minutes ago) <Sasha Klein>
bf14a52 - fix: more config (27 minutes ago) <Sasha Klein>
3052a8f - fix: set up base config (34 minutes ago) <Sasha Klein>
182ad3d - fix: try cltest again (19 hours ago) <Sasha Klein>
851a981 - fix: try cltest (19 hours ago) <Sasha Klein>
8dfec65 - fix: testing fix (19 hours ago) <Sasha Klein>
e266a14 - feat: testing feature (19 hours ago) <Sasha Klein>
953c5d2 - fix: updated package name (19 hours ago) <Sasha Klein>
f50c5e9 - feat: set up release please (19 hours ago) <Sasha Klein>
d1bb908 - docs: adding documentation on major bumps (20 hours ago) <Sasha Klein>
eee71e0 - docs: updated readme (20 hours ago) <Sasha Klein>
aba4104 - feat: adds commit linting (20 hours ago) <Sasha Klein>
39d4899 - (tag: v1.0.21, upstream/master, origin/master, master, cltest3, cltest2, cltest) 1.0.21 (22 hours ago) <Sasha Klein>
```

**Comment by chingor13** [comment]
Currently, it looks like the action is running against your `master` branch not your `commit-lint` branch (see `Looking for latest release on branch: master with prefix: my-package-name` log message). What are the commit messages on `master`? If you want it to look at the `commit-lint` branch, then set the [`default-branch` action option](https://github.com/google-github-actions/release-please-action#configuration).

Additionally, you are not setting [the `command` option to `manifest`](https://github.com/google-github-actions/release-please-action#the-command-option), so it is only looking at GitHub releases/tags and is ignoring the `release-please-config.json` and `.release-please-manifest.json` files.

**Comment by sashafklein** [comment]
Making those two changes fixed it! Thanks
